rules/windows/persistence_local_scheduled_task_scripting.toml

```diff
@@ -0,0 +1,40 @@
+[metadata]
+creation_date = "2020/11/29"
+maturity = "production"
+updated_date = "2020/11/29"
+
+[rule]
+author = ["Elastic"]
+description = "A scheduled task was created by a Windows script via cscript.exe, wscript.exe or powershell.exe, this can be used by an adversary to establish persistence."
```

Comment:
needs to be linted
```suggestion
description = "A scheduled task was created by a Windows script via cscript.exe, wscript.exe or powershell.exe. This can be abused by an adversary to establish persistence."
```